muxnect/server.py

```diff
@@ -144,13 +142,12 @@ def command_line():
     web_app = threading.Thread(target=app.run, kwargs=web_app_args)
 
     web_app.start()
-    print('Listening on {}'.format(url))
+    print('Listening on {0}'.format(url))
     print('Press CTRL+C to exit muxnect')
 
     if not detach:
         session.attach_session()
 
 
 if __name__ == '__main__':
-
```

Comment:
Seems to be a completely unnecessary empty line, along with 1 or 2 others :thinking:

Reply:
I've always wondered whether it looks good to have a blank line after `__name__ == '__main__'` but after some research, seems like I need to fix myself. Thanks!

Reply:
About the second index missing, I've simply overseen that and fixed it though. Thanks.
Regarding the blank line in the "`ifmain`" block, seems to be a question of general preference; even though I don't put one there myself and it does not seem very common, as you've noticed too. I'm curious what PEP8 would say about this one :smile: